PED/SetPedCanLosePropsOnDamage.md

```diff
@@ -0,0 +1,30 @@
+---
+ns: PED
+aliases: ["0xE861D0B05C7662B8"]
+---
+## SET_PED_CAN_LOSE_PROPS_ON_DAMAGE
+
+```c
+// 0xE861D0B05C7662B8 0x2F9550C2
+void SET_PED_CAN_LOSE_PROPS_ON_DAMAGE(Ped ped, BOOL p1, int p2);
+```
+
+It basically makes the ped lose (or not lose) their props (like glasses or helmets/hat) when someone punches or pushes the ped.
+I'm pretty sure this is what's being used in GTA:O to keep players from knocking other player's hats/glasses off when in combat.
+
+This is a toggle so no need to loop it every frame.
```

Comment:
`SET_` prefix implies this, `so no need to loop it` -> `so it doesn't need to be looped`?